docker/vttestserver/Dockerfile.mysql80

```diff
@@ -55,4 +55,4 @@ COPY --from=builder --chown=vitess:vitess /vt/install /vt
 VOLUME /vt/vtdataroot
 USER vitess
 
-CMD /vt/bin/vttestserver -port $PORT -keyspaces $KEYSPACES -num_shards $NUM_SHARDS -mysql_bind_host ${MYSQL_BIND_HOST:-127.0.0.1}
+CMD /vt/bin/vttestserver -port $PORT -keyspaces $KEYSPACES -num_shards $NUM_SHARDS -mysql_bind_host ${MYSQL_BIND_HOST:-127.0.0.1} -mysql_server_version 8.0.0-vitess
```

Comment:
We should probably use a current mysql version - say 8.0.21. 
To make this more usable, how about adding an env variable similar to the others and defaulting to 8.0.21-Vitess?

Reply:
Done